src/components/SignInOrJoinFree.js

```diff
@@ -40,7 +40,7 @@ class SignInOrJoinFree extends React.Component {
   };
 
   state = {
-    form: 'signin',
```

Comment:
I 100% agree with this change for the contribution flow, it makes a lot of sense.

However we also use this component in many other places, for example in `AuthenticatedPage` that you'll usually reach by clicking a link in an email. Users that arrive on this page always have an account and they'll always want to login, not create an account.

Can we have a props for this and limit the change for the contribution flow only?